### PR TITLE
wip: Generate theme section

### PR DESCRIPTION
**Pull request by ipeshev**
<a name"2.3.0"></a>
#### Bug Fixes

* add themes dependencies ([890cdf3d](https://github.com/telerik/kendo-angular-e2e/commit/890cdf3d))

#### Features

* test ([8628cf92](https://github.com/telerik/kendo-angular-e2e/commit/8628cf92))

#### Themes required

* @progress/kendo-theme-default: 4.19.0
* @progress/kendo-theme-material: 3.16.0
* @progress/kendo-theme-bootstrap: 4.17.0

### CODE REVIEW

**Comment by ipeshev** [comment]
Sample output from @progress/kendo-angular-e2e ( this will be the pilot project )

**Comment by ipeshev** [comment]
`Themes dependencies` is closer to the nature if the thing

**Comment by tsvetomir** [comment]
Moved to build tasks.

**Comment by ipeshev** [comment]
Reopening while confirm the other is working

File: generateNotesKendoAngular.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    let themesSection = "";
+
+    for (const pack in devDeps) {
+        if (pack.match(/@progress\/kendo-theme-/)) {
+            themesSection += `* ${pack}: ${devDeps[pack]}\n`;
+        }
+    }
+
+    if (themesSection.length) {
+        themesSection = "#### Themes supported\n\n" + themesSection;
```
Comment (Raisolution) [inline generateNotesKendoAngular.js:18]: "Themes supported" sounds strange. I would suggest to go for "Supported themes" or "Supported theme versions".

Comment (ipeshev) [reply generateNotesKendoAngular.js:18]: ... [moving here](https://github.com/telerik/kendo-build-tasks/pull/227) .

**Pull request closed without merging**